builder/azure/arm/config.go

```diff
@@ -1334,22 +1337,21 @@ func assertRequiredParametersSet(c *Config, errs *packersdk.MultiError) {
 		}
 		// Validate target region settings; it can be the deprecated replicated_regions attribute or multiple target_region blocks
 		if (len(c.SharedGalleryDestination.SigDestinationReplicationRegions) > 0) && (len(c.SharedGalleryDestination.SigDestinationTargetRegions) > 0) {
-			errs = packersdk.MultiErrorAppend(errs, errors.New("`replicated_regions` can not be defined alongside `target_region`; you can defined a target_region for each destination region you wish to replicate to."))
+			errs = packersdk.MultiErrorAppend(errs, errors.New("`replicated_regions` can not be defined alongside `target_region`; you can define a target_region for each destination region you wish to replicate to."))
+		}
+
+		if (len(c.SharedGalleryDestination.SigDestinationTargetRegions) > 0) && c.SharedGalleryImageVersionReplicaCount != 0 {
+			errs = packersdk.MultiErrorAppend(errs, errors.New("`shared_image_gallery_replica_count` can not be defined alongside `target_region`; you can specify the number of replicas per region within target_region blocks."))
```

Comment:
Two notes on this I found the error message a bit confusing when talking about `single target_region blocks` so I reworded that

worth noting this is a breaking change since users would have had to set the replica count with shared_image_gallery_replica_count before, even when using a target_region block, so its worth making the next release a minor version bump, and calling this out in the release notes I think